packages/@lightningjs/ui-components/src/components/Column/Column.mdx

```diff
@@ -121,11 +121,7 @@ If both of those properties are enabled, `alwaysScroll` will take precedence ove
 
 ### Style Properties
 
-The Column component shares a few of the same style properties as the ones listed in NavigationManager. For a detailed description, please refer to style properties section in the [NavigationManager component](?path=/docs/components-navigationmanager--docs)
-
-scrollIndex: number
-itemSpacing: number
-itemTransition: <DocsLink id="lng.Transition" />
+The Column component shares the same style properties as the ones listed in NavigationManager. For a detailed description, please refer to style properties section in the [NavigationManager component](?path=/docs/components-navigationmanager--docs)
```

Comment:
it might be nice to include the style properties hash to take devs straight to the section we're referencing here

```suggestion
The Column component shares the same style properties as the ones listed in NavigationManager. For a detailed description, please refer to style properties section in the [NavigationManager component](?path=/docs/components-navigationmanager--docs#style-properties)
```